array_api_compat/common/_helpers.py

```diff
@@ -204,6 +231,12 @@ def _torch_to_device(x, device, /, stream=None):
         raise NotImplementedError
     return x.to(device)
 
+def _jax_to_device(x, device, /, stream=None):
+    import jax
+    if stream is not None:
+        raise NotImplementedError
+    return jax.device_put(x, device)
```

Comment:
Is this helper function for `to_device` correct?

Reply:
Yes, `device_put` would be the right method here.